lib/acl.c

```diff
@@ -125,9 +144,16 @@ struct rte_acl_field_def ipv6_defs[NUM_FIELDS_IPV6] = {
 	},
 };
 
+/*
+ * For each ACL rule set, register a match function that parses
+ * the unmatched IPv6 packets, and direct them to the corresponding
+ * blocks or drop them. This functionality is for the ext_cb_f parameter
+ * and that it's necessary because of variable IP headers that
+ * may not match the ACLs.
+ */
```

Comment:
Please add the following warning to the comment above:
```
 * WARNING
 *   You must only register filters that are not subject to
 *   the control of attackers. Otherwise, attackers can overwhelm
 *   Gatekeeper servers since the current implementation of these filters
 *   is not very efficient due to the variable header of IP.
```